docs/user/custom-domains.rst

```diff
@@ -2,125 +2,41 @@ Custom Domains
 ==============
 
 Custom domains allow you to serve your documentation from your own domain.
+For instance, this could be ``docs.example.com``.
 This is great for maintaining a consistent brand for your documentation and application.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+*By default*, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
-For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
+How does it work?
+-----------------
 
-.. contents:: Contents
-    :local:
+To use a custom domain, you enter the domain in your Read the Docs project's administration and update your DNS provider with a new DNS entry.
```

Comment:
I think we normally say `:guilabel:Admin` or `Dashboard`: https://docs.readthedocs.io/en/stable/glossary.html#term-dashboard -- but we should standardize on it.

Reply:
I opted for the "Dashboard" term -- the :guilabel: might make the "genre" blurry.

Reply:
Dashboard and Admin are two different things.

The Dashboard is the "whole readthedocs.org application when the user is logged in". The Admin is the "project's admin page"

In this case, it seems the correct would be `` :guilabel:`Admin` `` --since the user has to add this domain in the admin of the project.

Reply:
Actually, if you go to the current definition of it, it does not makes sense in this context:

![Screenshot_2022-11-30_19-37-02](https://user-images.githubusercontent.com/244656/204880919-d9305590-be7a-49bb-9a48-8bc9e9ea6163.png)

Reply:
Hmm, I see your point @humitos - I have misunderstood what @ericholscher was saying as two equal alternatives.

What I don't like so much is to put `:guilabel:` in the explanation because it indicates a concrete instruction. That's a widget we would normally put in a howto. In an explanation, it's distracting.

Is it possible that we add "project administration" or "project admin" to the glossary?

Reply:
Btw I really like to see a `:guilabel:`.. but in a How-To, Reference or Tutorial :) I'm afraid that it's a gateway drug or slippery slope for an explanation to end up doing how-to.

Reply:
I think telling people *where* things are is definitely explanation, and well-suited for `:guilabel:`

Reply:
Alright, it's a good case to close a comment thread :100: I don't mind it really being `:guilabel:`, so let's do that and we're definitely gonna have a good documentation article :+1:

Reply:
We have been using `:guilabel:` when we are talking directly about a link/tab/something in particular from the UI. When we are talking about a "general concept" we could omit the `:guilabel:`. We can change that behavior if we consider it's not accurate, but probably is another conversation 😄